_quarto.yml

```diff
@@ -3,8 +3,7 @@ project:
   output-dir: docs
 
 website:
-  title: HERA
-  subtitle: Health Economics in R for All
```

Comment:
this isn't a valid property, see https://github.com/HTAhackathon2024/HERA/actions/runs/10267254875/job/28407373921

Reply:
Right - removed this #42